### DISCUSSION TITLE
How to modify the color of filepath?

### DISCUSSION THREAD

**Discussion by ycycyyc** [ibhagwan/fzf-lua#1219] [Q&A] [answered]
Is there a way to change the color of the file path when I find the references?
<img width="556" alt="image" src="https://github.com/ibhagwan/fzf-lua/assets/30541266/0fe0da87-dc03-4d44-95d7-ab43e6261a01">

maybe like this:
<img width="566" alt="image" src="https://github.com/ibhagwan/fzf-lua/assets/30541266/559d5c1e-7d7b-4206-9eae-051d4e46514f">

**Comment by ibhagwan** [comment, marked as answer]
You can change the color of the whole line with `fzf_colors` but not the part only.

You can craft your own references picker with fzf_exec if you want to do that.

**Reply by ycycyyc** [reply]
Thanks a lot.
